src/hooks/useAccountENSDomainGroups.ts

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import useAccountENSDomains from './useAccountENSDomains';
+import useAccountProfile from './useAccountProfile';
+
+export default function useAccountENSDomainGroups() {
```

Comment:
maybe we should just put this stuff in `useAccountENSDomains` to prevent hook fatigue?